bitnami/external-dns/README.md

```diff
@@ -262,6 +262,9 @@ This chart allows you to set your custom affinity using the `affinity` parameter
 
 As an alternative, you can use of the preset configurations for pod affinity, pod anti-affinity, and node affinity available at the [bitnami/common](https://github.com/bitnami/charts/tree/master/bitnami/common#affinities) chart. To do so, set the `podAffinityPreset`, `podAntiAffinityPreset`, or `nodeAffinityPreset` parameters.
 
+### Using IRSA 
+If you are deploying to AWS EKS and you want to leverage [IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html). You will need to override `fsGroup` and `runAsUser` with `65534` and `0` respectively. Otherwise service account token will not be properly mounted.
+
```

Comment:
it could be interesting to add helm parameter or yaml example that specifies the actual path like:

```
--set path.to.fsGroup=65534 --set path.to.runAsUser=0
```

Reply:
Added